tests/components/plugwise/test_init.py

```diff
@@ -67,7 +71,10 @@ async def test_config_entry_not_ready(
     await hass.async_block_till_done()
 
     assert len(mock_smile_anna.connect.mock_calls) == 1
-    assert mock_config_entry.state is ConfigEntryState.SETUP_RETRY
+    assert mock_config_entry.state in (
+        ConfigEntryState.SETUP_ERROR,
+        ConfigEntryState.SETUP_RETRY,
+    )
```

Comment:
This is not specific enough. This test can now be wrong without noticing. Avoid the use of `in` here.

Reply:
Yes, that works. Alternatively (and probably easier), add the expected result to the parameterize decorator of this testing method.

Reply:
Implemented as suggested.